cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -0,0 +1,1298 @@
+                    ]:
+                        self.skipTest("Format is fail")
+
+                    for include_images in (False, True):
+                        # create task
+                        images = self._generate_task_images(3)
+                        if dump_format_name == "Market-1501 1.0":
+                            task = self._create_task(tasks["market1501"], images)
+                        elif dump_format_name in ["ICDAR Localization 1.0",
+                                "ICDAR Recognition 1.0"]:
+                            task = self._create_task(tasks["icdar_localization_and_recognition"], images)
+                        elif dump_format_name == "ICDAR Segmentation 1.0":
+                            task = self._create_task(tasks["icdar_segmentation"], images)
+                        else:
+                            task = self._create_task(tasks["main"], images)
+
+                        # create annotations
+                        if dump_format_name in [
+                            "MOT 1.1", "MOTS PNG 1.0", \
+                            "PASCAL VOC 1.1", "Segmentation mask 1.1", \
+                            "TFRecord 1.0", "YOLO 1.1", "ImageNet 1.0", \
+                            "WiderFace 1.0", "VGGFace2 1.0", \
+                        ]:
+                            self._create_annotations(task, dump_format_name, "default")
+                        else:
+                            self._create_annotations(task, dump_format_name, "random")
+
+                        task_id = task["id"]
+                        data_from_task_before_upload = self._get_data_from_task(task_id, include_images)
+
+                        # dump annotations
+                        url = self._generate_url_dump_tasks_annotations(task_id, dump_format_name)
+                        with TestDir() as test_dir:
+                            file_zip_name = osp.join(test_dir, f'{test_name}_{dump_format_name}.zip')
+                            data = {
+                                "format": dump_format_name,
+                            }
+                            response = self._dump_api_v1_tasks_id_annotations(url, data, self.admin)
+                            self.assertEqual(response.status_code, status.HTTP_202_ACCEPTED)
+                            response = self._dump_api_v1_tasks_id_annotations(url, data, self.admin)
+                            self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+                            data = {
+                                "format": dump_format_name,
+                                "action": "download",
+                            }
+                            response = self._dump_api_v1_tasks_id_annotations(url, data, self.admin)
+                            self.assertEqual(response.status_code, status.HTTP_200_OK)
+                            if response.status_code == status.HTTP_200_OK:
```

Comment:
There could be something other than 200?

Reply:
If user is None it returns code is 401.

Reply:
I mean, on the previous line, this is already checked. If the `status code != 200`, it will not reach here.

Reply:
Removed.